do_all_tables.sh

```diff
@@ -53,6 +53,9 @@ do
 		arch=armoabi	extraflags=				generate_table
 		arch=arm	extraflags=-D__ARM_EABI__		generate_table
 		;;
+	loongarch)
+		arch=loongarch64	extraflags=-D_LOONGARCH_SZLONG=64	generate_table
```

Comment:
So is it loongarch or loongarch64?

32/64 is added to architecture names only if there are two of them or when kernel name contain them.

Reply:
Ah, I see that you consider 32bit arch as well. OK.